### ISSUE TITLE
Polish translation

### ISSUE THREAD

**Issue by mayro57** [RedChimera/IWD2EE#123]
The installation of the mod obscures much of the Polish language version. Could something be done to keep the Polish language entirely where it is?

**Comment by WillScarlettOhara** [comment]
No, many of the game's components are modified and others are even added. It would be worse to keep false item, spell or class descriptions in Polish than to have the new descriptions for these items, spells or classes. There are two solutions: install only the core component and a few others that don't modify the game data like xp modifications or sneak attack delay removal, or start a mod translation from the files in the tra/english/ folder. ;}

**Comment by mayro57** [comment]
Thank you for explaining, have a nice day 😊
